dts/bindings/pwm/nordic,nrf-sw-pwm.yaml

```diff
@@ -0,0 +1,65 @@
+---
+title: nRF SW PWM
+version: 0.1
+
+description: >
+    This binding gives a base representation of the nRFx S/W PWM
+
+properties:
+    compatible:
+      constraint: "nordic,nrf-sw-pwm"
+
+    label:
+      type: string
+      category: required
+      description: Human readable string describing the device (used by Zephyr for API name)
+      generation: define
+
+    ch0-pin:
```

Comment:
This should be removed, as well as `ch1-pin` and `ch2-pin`.

Reply:
Yes